tests/test_session.py

```diff
@@ -329,3 +334,279 @@ async def get_session_information(
     data = await my_session.get_session_data()
 
     assert data == {"foo": "bar"}
+
+
+from supertokens_python.recipe.session.exceptions import raise_unauthorised_exception
+from supertokens_python.recipe.session.interfaces import APIInterface, APIOptions
+from tests.utils import extract_all_cookies, get_st_init_args
+
+
+async def test_revoking_session_during_refresh_with_revoke_session_with_200(
+    driver_config_client: TestClient,
+):
+    def session_api_override(oi: APIInterface) -> APIInterface:
+        oi_refresh_post = oi.refresh_post
+
+        async def refresh_post(api_options: APIOptions, user_context: Dict[str, Any]):
+            s = await oi_refresh_post(api_options, user_context)
+            await s.revoke_session()
+            api_options.response.set_status_code(200)  # type: ignore
+            api_options.response.set_html_content("")  # type: ignore
```

Comment:
no need to do this.. it happens on it's own anyway

Reply:
done.